tests/objc-cli/test-managed.m

```diff
@@ -22,17 +22,21 @@ int main ()
 		NSLog (@"%s instance property getter", [query answer] == 42 ? "[PASS]" : "[FAIL]");
 		query.answer = 911;
 		NSLog (@"%s instance property setter", [query answer] == 911 ? "[PASS]" : "[FAIL]");
+		[query release];
```

Comment:
Seems that we can [1] and I think we should - just to cut down on support issues and the test matrix.

[1] http://tewha.net/2012/09/make-your-library-enforce-arc/